src/Module/User/Import.php

```diff
@@ -0,0 +1,415 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2022, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\User;
+
+use Friendica\App;
+use Friendica\Core\Config\Capability\IManageConfigValues;
+use Friendica\Core\L10n;
+use Friendica\Core\PConfig\Capability\IManagePersonalConfigValues;
+use Friendica\Core\Protocol;
+use Friendica\Core\Renderer;
+use Friendica\Core\System;
+use Friendica\Core\Worker;
+use Friendica\Database\Database;
+use Friendica\Database\DBA;
+use Friendica\Database\DBStructure;
+use Friendica\Model\Photo;
+use Friendica\Model\Profile;
+use Friendica\Module\Response;
+use Friendica\Navigation\SystemMessages;
+use Friendica\Network\HTTPException;
+use Friendica\Object\Image;
+use Friendica\Security\PermissionSet\Repository\PermissionSet;
+use Friendica\Util\Profiler;
+use Friendica\Util\Strings;
+use Friendica\Worker\Delivery;
+use Psr\Log\LoggerInterface;
+
+class Import extends \Friendica\BaseModule
+{
+	const IMPORT_DEBUG = false;
```

Comment:
I think this is a weird way of debugging enable triggering, it's the first time we do it that code invasive.. maybe a parameter instead in the constructor `bool $dryRun`?

Reply:
Or specifying the corresponding methods protected and override them for testing... Seems much cleaner and like the way we do it at other places

Reply:
I would remove the whole thing, but leaving it as is was easier in this process.

Reply:
So do you plan to adapt it or leave it as is?

Reply:
Leave it for now until we can fully test.